packages/wdio-allure-reporter/src/index.js

```diff
@@ -110,8 +110,11 @@ class AllureReporter extends WDIOReporter {
         const currentTest = this.allure.getCurrentTest()
 
         if (!this.isMultiremote) {
-            const { browserName, deviceName } = this.capabilities
-            const targetName = browserName || deviceName || cid
+            const { browserName, deviceName, desired } = this.capabilities
+            let targetName = browserName || deviceName || cid
+            if(desired && desired.deviceName && desired.platformVersion){
+                targetName = desired.deviceName+'-'+desired.platformVersion
```

Comment:
How about we separate by space
```suggestion
                targetName = `${desired.deviceName} ${desired.platformVersion}`
```